src/page-multi-facility/FacilityRow.tsx

```diff
@@ -109,11 +140,18 @@ const FacilityRow: React.FC<Props> = ({
   const [model, updateModel] = useModel();
 
   const { rtData, setFacility } = useContext(FacilityContext);
+
   const [facility, updateFacility] = useState(initialFacility);
-  let useRt, facilityRtData;
+  let useRt,
+    facilityRtData,
+    latestRt = null;
   if (useFlag(["useRt"])) {
     useRt = true;
-    facilityRtData = rtData ? rtData[facility.id] : undefined;
+    facilityRtData = rtData ? rtData[facility.id] : null;
```

Comment:
i don't think this is correct ... if the fetch has failed this value will already be null. if it's undefined that generally means a fetch has not been completed yet. Similarly, `rtData` should only really ever be undefined while the app is initializing and we haven't done the initial fetch yet (a "loading" state for practical purposes). 

I know it's not very explicit but we should try to stick to that general convention if possible to avoid making it even more confusing than it already is.

Reply:
Updated